### PR TITLE
fix is_armable can not handle 'Check mag field'

### PR DESCRIPTION
**Pull request by Huibean**
I found that the dronekit is_armable api can not handle situation when 'Check mag field' appear with Geomagnetic interference，add this pr to cover that situation

### CODE REVIEW

File: dronekit/__init__.py [hunk 1]
```diff
@@ -1539,7 +1543,7 @@ def flush(self):
             :py:func:`Vehicle.commands.upload() <Vehicle.commands.upload>`.
         """
         return self.commands.upload()
-
```
Comment (amilcarlucas) [inline dronekit/__init__.py]: Please remove this whitespace change from the PR

**Comment by Huibean** [comment]
ok, fix that

**Comment by amilcarlucas** [comment]
ca you please squash the two commits and do a force push ?

**Comment by Huibean** [comment]
sorry for that, changed

**Comment by amilcarlucas** [comment]
It needs a rebase

**Comment by Huibean** [comment]
Hi @amilcarlucas, I am closing this pr cause it's been a while I can not edit on it now

**Pull request closed without merging**